### ISSUE TITLE
Cannot Enter Space in Namespace Filter

### ISSUE THREAD

**Issue by momesgin** [rancher/dashboard#14425]
**Setup**
- Rancher version: latest
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

Entering space is blocked in the Namespace filter. 

**To Reproduce**

Go to Explorer => Open Namespace filter dropdown on the top => Enter space

**Result**
Entering spaces is blocked
**Expected Result**
Entering spaces should not be blocked

**Screenshots**

**Additional context**

Whoever takes this issue to work on might also consider looking for the other cases in the app where `@keydown.space.prevent` is used
